api/integrations/lead_tracking/pipedrive/lead_tracker.py

```diff
@@ -49,7 +50,7 @@ def should_track(user: FFAdminUser):  # type: ignore[no-untyped-def]
 
         return True
 
-    def create_lead(self, user: FFAdminUser) -> PipedriveLead:
+    def create_organisation_lead(self, user: FFAdminUser) -> PipedriveLead:
```

Comment:
My expectation from this latest set of changes was that this method would be removed / reverted to just be `create_lead` again?

Reply:
resolved in #5626